proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/logging/EidasResponseAttributesHashLoggerHelper.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.ida.notification.translator.logging;
+
+import uk.gov.ida.notification.contracts.verifyserviceprovider.Attribute;
+import uk.gov.ida.notification.contracts.verifyserviceprovider.Attributes;
+import uk.gov.ida.saml.core.transformers.EidasResponseAttributesHashLogger;
+
+public final class EidasResponseAttributesHashLoggerHelper {
+
+    EidasResponseAttributesHashLogger eidasResponseAttributesHashLogger;
+
+    private EidasResponseAttributesHashLoggerHelper() {}
+
+    public EidasResponseAttributesHashLoggerHelper(EidasResponseAttributesHashLogger eidasResponseAttributesHashLogger) {
+        this.eidasResponseAttributesHashLogger = eidasResponseAttributesHashLogger;
+    }
+
+    public EidasResponseAttributesHashLogger applyAttributesToHashLogger(Attributes attributes, String pid) {
+
+        eidasResponseAttributesHashLogger.setPid(pid);
+
+        if (attributes != null) {
+            if (attributes.getFirstNames() != null) {
+                attributes.getFirstNames().stream()
+                        .filter(Attribute::isVerified)
+                        .findFirst()
+                        .ifPresent(firstName -> eidasResponseAttributesHashLogger.setFirstName(firstName.getValue()));
+            }
+            if (attributes.getMiddleNames() != null) {
```

Comment:
`attributes.getMiddleNames()` is never null so no need for null guard. Preferably also guarantee `getFirstNames()` and other list getters are non null, so this reads easier? Not a deal breaker but makes me happy

Reply:
Modified Attributes to guarantee non null and have removed the guards.